tests/test_mvkde.py

```diff
@@ -219,7 +219,8 @@ def test_univariate_two_sample(i):
     ax.grid()
```

Comment:
Writing `_, pvalue = ks_2samp(...)` vs `pvalue = ks_2samp(...)[1]` doesn't make much difference.  The problem I was avoiding in this file is that `pvalue = ks_2samp(...).pvalue` didn't work until more recent versions of scipy, and I just made it consistent throughout the file.  We can change the other file too -- I just didn't notice it.